main/global_state.h

```diff
@@ -37,6 +37,7 @@ typedef struct
     uint64_t shares_accepted;
     uint64_t shares_rejected;
     uint64_t work_received;
+    int64_t last_notify_timestamp;
```

Comment:
Minor thing: field is only used in stratum_task, so it can just be a static field in there.